assets/css/edd-admin.css

```diff
@@ -3981,3 +3939,78 @@ div:not(#edd-item-tab-wrapper) + #edd-item-card-wrapper .item-section { /** [1]
 	font-style: italic;
 	width: 150px;
 }
+
+/**
+ * Forms
+ * ---------------------------------------------------------- */
+
+/**
+ * Form Group
+ *
+ * <div class="edd-form-group">
+ *   <label class="edd-form-group__label">Label</label>
+ *   <div class="edd-form-group__control">
+ *   	<input type="text" class="edd-form-group__input" />
+ *   </div>
+ *   <p class="edd-form-group__help">Help</p>
+ * </div>
+ *
+ *
+ * <fieldset class="edd-form-group">
+ *   <legend class="edd-form-group__label">Label</label>
+ *
+ *   <div class="edd-form-group__control">
+ *   	<input type="radio" class="edd-form-group__input" />
+ *   	<label>Radio 1</label>
+ *   </div>
+ *
+ *   <div class="edd-form-group__control">
+ *   	<input type="radio" class="edd-form-group__input" />
+ *   	<label>Radio 2</label>
+ *   </div>
+ * </fieldset>
+ *
+ */
+.edd-form-group {
+	margin-bottom: 12px;
```

Comment:
With the `8px` under the labels, I think this needs to be increased to `16px` or `18px`. Thoughts @LisaCee @robincornett?

Reply:
I'm always a fan of white space. Could we also consider removing the `:last-of-type` modifier on it, at least to experiment? I'm seeing that applied more places than I expect.

Reply:
@robincornett Where is it causing issues?

Reply:
With the way the `form-group`s are organized, I think most of them fall into the `:last-of-type` category.

Reply:
Everything here is `:last-of-type`, with `margin-bottom:0` :
![image](https://user-images.githubusercontent.com/18077352/80236534-0fbf1080-8610-11ea-9ca3-38b010c8f5c0.png)

vs without `:last-of-type` and current `.edd-form-group {margin-bottom: 12px}`:

![image](https://user-images.githubusercontent.com/18077352/80236715-5f9dd780-8610-11ea-8192-67f39cba7e90.png)

Reply:
This is what I'm seeing as well -- removing `:last-of-type` will add a lot of extra space.

Reply:
Yes, thanks @LisaCee, that is primarily where I was feeling it.

Really, the only place right now where I could see it actually making a helpful difference to have the `:last-of-type` modifier is at the end of the address block field, but it actually does nothing there because the section content has this rule:

```css
.edd-sections-wrap .section-wrap .section-content > * {
    margin: 20px;
}
```
So removing the margin from the last-of-type wouldn't even affect that.

Reply:
@robincornett I think @LisaCee was showing the opposite. Currently it applies to all items in "Order Attributes" and "Order Extras", which is correct:

**With**
<img width="481" alt="Screen Shot 2020-04-24 at 1 16 37 PM" src="https://user-images.githubusercontent.com/491124/80239263-ea40ff80-862d-11ea-8356-f9af1c092fa7.png">

**Without**
<img width="489" alt="Screen Shot 2020-04-24 at 1 16 46 PM" src="https://user-images.githubusercontent.com/491124/80239266-eb722c80-862d-11ea-9be4-3cb69db36963.png">

Reply:
What about changing the css for `.edd-form-group__control:last-of-type`?  
Current
![image](https://user-images.githubusercontent.com/18077352/80239953-ef925000-8615-11ea-8b93-260a10bfc078.png)

vs without
![image](https://user-images.githubusercontent.com/18077352/80240039-194b7700-8616-11ea-898c-aeec453daae0.png)

That gives it a bit more spacing at the bottom without making it really big.

Reply:
@LisaCee If we want more space in those rows we should adjust the padding on `.edd-admin-box-inside` instead.

Let's do this:

- Change `.edd-form-group` to have `16px` bottom margin.
- Change `.edd-admin-box-inside` to have `12px` padding all around (increasing 2px on the top and bottom from previous)